packages/kbn-interpreter/package.json

```diff
@@ -1,12 +1,11 @@
-{
+
+
+  {
   "name": "@kbn/interpreter",
   "private": "true",
   "version": "1.0.0",
   "license": "SSPL-1.0 OR Elastic License 2.0",
   "scripts": {
-    "interpreter:peg": "../../node_modules/.bin/pegjs src/common/lib/grammar.peg",
-    "build": "node scripts/build",
-    "kbn:bootstrap": "node scripts/build --dev",
-    "kbn:watch": "node scripts/build --dev --watch"
+    "interpreter:peg": "../../node_modules/.bin/pegjs src/common/lib/grammar.peg"
```

Comment:
can we run this as part of bootstrap ?

Reply:
@jbudz probably in order to accomplish this we need to do something like https://github.com/elastic/kibana/blob/master/packages/kbn-tinymath/BUILD.bazel#L33

Reply:
except that interpreter is not using peggy yet but still the old pegjs.
i am ok with moving to peggyjs in the same pr as long as it doesn't break